src/Avalonia.Controls/ComboBox.cs

```diff
@@ -184,6 +184,25 @@ protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
             this.UpdateSelectionBoxItem(SelectedItem);
         }
 
+        // Because the SelectedItem isn't connected to the visual tree
+        public override void InvalidateFlowDirection()
```

Comment:
Can we rename this to `InvalidateMirrorTransform`? The FlowDirection isn't changed by this method but the mirror transform.

Reply:
I know, I thought IsMirrorTransform should not only be dictated by FlowDirection, but you seem to be right.

Will I change?

Reply:
Y please rename it

Reply:
ok